tests/unit/remote/gdrive/test_gdrive.py

```diff
@@ -0,0 +1,16 @@
+import mock
+from dvc.remote.gdrive import RemoteGDrive
+
+
+@mock.patch("dvc.remote.gdrive.RemoteGDrive.init_drive")
+def test_init_drive(repo):
+    url = "gdrive://root/data"
+    gdrive = RemoteGDrive(repo, {"url": url})
+    assert str(gdrive.path_info) == url
+
+
+@mock.patch("dvc.remote.gdrive.RemoteGDrive.init_drive")
+def test_init_folder_id(repo):
```

Comment:
how is it different from the previous test?

Reply:
Have removed this for now, thanks